src/client/components/ThreadDetails.tsx

```diff
@@ -333,11 +333,15 @@ const SeparatorLine = styled.hr`
 const StyledThreadByID = styled(experimental.Thread.ByID)`
   &.cord-component {
     border: none;
+    height: 100%;
   }
   &.cord-component .cord-composer {
     margin: 0 20px 20px;
   }
   &.cord-component .cord-thread-header-container {
     display: none;
   }
+  &.cord-component .cord-scroll-container {
+    max-height: none;
+  }
```

Comment:
Oh interesting, the `scroll-container` shouldn't have a `max-height` (it used to, but newer version don't) -- is it Clack setting it?

Reply:
Trying to find where it is coming from, but it doesn't look like it's coming from clack <img width="466" alt="Screenshot 2024-04-24 at 14 00 17" src="https://github.com/getcord/clack/assets/102948032/26fca19b-e196-46f3-909a-8bee17581796">

Reply:
Oh, I see CSS version is 1.41.0 -- updating to 1.42 will fix it